tests/baselines/reference/wasmImportsSimple.js

```diff
@@ -0,0 +1,15 @@
+//// [wasmImportsSimple.ts]
+import * as mod from "/.lib/test.wasm";
+
+let running = 0;
+for (let i = 0; i < 10; i++) {
+    running += mod.addTwo(i, i);
+}
+
+
+//// [wasmImportsSimple.js]
+import * as mod from "test.wasm";

```

Comment:
The emitted module specifier doesn’t look right.

Reply:
The test harness strips the leading `/.lib/` because it's on the real disk the machine is running on (and has the corresponding real path) - it only does this in the test harness~